packages/@aws-cdk/aws-servicediscovery/lib/private-dns-namespace.ts

```diff
@@ -0,0 +1,57 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import cdk = require('@aws-cdk/cdk');
+import { CfnPrivateDnsNamespace} from './servicediscovery.generated';
+
+export interface PrivateDnsNamespaceProps {
+  /**
+   * A name for the HttpNamespace.
+   *
+   * @default CloudFormation-generated name
+   */
+  name: string;
+
+  /**
+   * A description of the namespace.
+   */
+  vpc: ec2.IVpcNetwork;
+
+  /**
+   * A description of the namespace.
+   */
+  description?: string;
+}
+
+/**
+ * Define a Service Discovery HTTP Namespace
+ */
+export class PrivateDnsNamespace extends cdk.Construct {
```

Comment:
Is there are more generic `INamespace` interface to be extracted between `HttpNamespace` and `PrivateDnsNamespace`, or does that not make sense?